src/include/verbose.h

```diff
@@ -163,14 +163,26 @@ struct __wt_verbose_multi_category {
 
 /*
  * __wt_verbose --
- *     Display a verbose message using the default verbosity level. Not an inlined function because
- *     you can't inline functions taking variadic arguments and we don't want to make a function
- *     call in production systems just to find out a verbose flag isn't set. The macro must take a
- *     format string and at least one additional argument, there's no portable way to remove the
- *     comma before an empty __VA_ARGS__ value.
+ *     Display a verbose message using the default verbosity level unless time out logs have been
+ *     requested for specific categories. Not an inlined function because you can't inline functions
+ *     taking variadic arguments and we don't want to make a function call in production systems
+ *     just to find out a verbose flag isn't set. The macro must take a format string and at least
+ *     one additional argument, there's no portable way to remove the comma before an empty
+ *     __VA_ARGS__ value.
  */
-#define __wt_verbose(session, category, fmt, ...) \
-    __wt_verbose_level(session, category, WT_VERBOSE_LEVEL_DEFAULT, fmt, __VA_ARGS__)
+#define __wt_verbose(session, category, fmt, ...)                                                 \
```

Comment:
I am concerned that each verbose call is going to pay the cost of the additional checks now. Earlier __wt_verbose_level was a single "if" check, now there are 9+. It will be great if we can find an approach where the call point to verbose doesn't involve additional checks.

Reply:
That will be prefered. The challenge would be to reset the level back to the former level, past the event you want to log debug messages for.

Reply:
I don't think we need to reset to the previous level as we log stuff 20ms before the crash.

Reply:
Not in this case of cache stuck, but for the slow checkpoint, or when eviction is too slow to drain. The cases where you want to restore to the previous level once the event is over.

Reply:
I believe it is the same for the slow checkpoint, or when eviction is too slow to drain. If we haven't made any progress in the last 3 minutes, it is unlikely we will get unstuck in the last 20ms before we time out. I could reduce this time if you think I should.
@sueloverso, what do you think?